paramiko/config.py

```diff
@@ -80,6 +81,27 @@ def parse(self, file_obj):
                 # at the end (for compatibility with issue #415). After 3.x, it
                 # will simply not get stripped, leaving a nice explicit marker.
                 host['config'][key] = None
+            elif key == 'include':
+                # support for Include directive in ssh_config
+
+                path = value
+                # according to SSH documentation if the path is relative, we look forward in ~/.ssh
+                if path[0] != '/' and path[0]!='~':
```

Comment:
you may want to use `os.path.isabs()` for the first part of this condition, to perhaps enable better compatibility with windows